src/Conf.ml

```diff
@@ -808,3 +823,13 @@ let action =
 and debug = !debug
 
 let parse_line_in_attribute = parse_line ~from:`Attribute
+
+let line_endings c file =
+  match c.line_endings with
+  | `Unix -> `Unix
+  | `Windows -> `Windows
+  | `Preserve -> (
+      let read_line = Stdio.In_channel.input_line ~fix_win_eol:false in
+      match Stdio.In_channel.with_file ~binary:false file ~f:read_line with
```

Comment:
Shouldn't binary be true

Reply:
@gpetiot did you see this? Have you tested on a windows machine?

Reply:
I don't have a windows machine at hand to test this.
I have no clue for the binary mode opening, if you are confident that it should be true I will change it.

Reply:
In non-binary (that is, text) mode, the runtime system on windows will helpfully translate \r\n to \n. So without ~binary:true, this code will never see a \r